### PR TITLE
Update to new standard

### PR DESCRIPTION
**Pull request by mueschm**
## Overview
Update to the new standard. This project mostly already conformed, so just updated some of the styling and the readme.

## Changes
1. Remove the big block of text on the site
2. Show movie form by default
3. Updated readme

### Local

chrome_EyyWtiUDpN

### Dev

chrome_6kGIaaWlnu

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,54 +16,113 @@
...
-</p>
-
 ---
 
-# ASP.net Core Starter Project
-[ASP.net](https://asp.net/) is a popular web framework for handling both the frotned and backend of your application. This repository contains a small working ASP.net web server and api that is packaged into an Architect Component.
 
-This starter application will show how easy it is to deploy an application both
-locally and in a remote environment.
+# Go Starter Project
```
Comment (ryan-cahill) [inline README.md:22]: I hadn't realized that C# code looked so much like Go code

File: README.md [hunk 2]
```diff
@@ -16,54 +16,113 @@
...
-[sign up for a free account](https://cloud.architect.io/signup).
+When this command completes, you can reach your new application by going to https://app.localhost.architect.sh.
+### Make your own changes
+
+This application's `architect.yml` file contains a `debug` block that enables hot-reloading for each service
+within the component. That means you can make changes to the source code and those changes will be applied to the
+environment automatically. This allows you to quickly iterate and see your changes without having to restart the
+application stack.
+
+Give it a try! Search inside your project for “Favorite Movie” and change this string to "Favorite Pizza" Once you save
```
Comment (ryan-cahill) [inline README.md:87]: The "Favorite movie" header mentioned here isn't included in the template

**Review by ryan-cahill — CHANGES_REQUESTED** [review]
Why is there an item seeded in the DB when the app starts? I think that could be confusing to some users

**Comment by ryan-cahill** [comment]
Also in testing the app (noticed with running `architect dev --debug=false` if you submit the form, then refresh the page after, it keeps submitting the form over and over on page refresh even though there's also nothing in the text fields

**Comment by mueschm** [comment]
Updated
image

**Comment by ryan-cahill** [comment]
Thanks for fixing those things. Two things left:

1. We added a DB liveness probe to the other templates which should be added here too
```
    liveness_probe:
      command: pg_isready -d ${{ secrets.database_name }} -U ${{ secrets.postgres_username }}
      interval: 10s
      failure_threshold: 3
```
2. I still think it's weird that there's an item seeded in the database when the app starts. It's even weirder that the rating is 10 when it's only supposed to be 1-5 according to the text field's placeholder

**Review by ryan-cahill — CHANGES_REQUESTED** [review]
Thanks for fixing those things. Three things left:

1. We added a DB liveness probe to the other templates which should be added here too
```
    liveness_probe:
      command: pg_isready -d ${{ secrets.database_name }} -U ${{ secrets.postgres_username }}
      interval: 10s
      failure_threshold: 3
```
2. I still think it's weird that there's an item seeded in the database when the app starts. It's even weirder that the rating is 10 when it's only supposed to be 1-5 according to the text field's placeholder
3. The app doesn't deploy to a remote environment and fails with the following error
```
warn: Microsoft.AspNetCore.DataProtection.Repositories.FileSystemXmlRepository[60]
Storing keys in a directory '/root/.aspnet/DataProtection-Keys' that may not be persisted outside of the container. Protected data will be unavailable when container is destroyed.
warn: Microsoft.AspNetCore.DataProtection.KeyManagement.XmlKeyManager[35]
No XML encryptor configured. Key {a3577738-831d-47d3-8456-2d79191adfcc} may be persisted to storage in unencrypted form.
Finished configuring web server
info: Microsoft.Hosting.Lifetime[14]
Now listening on: http://[::]:5001
info: Microsoft.Hosting.Lifetime[0]
Application started. Press Ctrl+C to shut down.
info: Microsoft.Hosting.Lifetime[0]
Hosting environment: Production
info: Microsoft.Hosting.Lifetime[0]
Content root path: /app
info: System.Net.Http.HttpClient.api.LogicalHandler[100]
Start processing HTTP request GET https://api.example-environment.architect-ryan.dev.arcitect.io/movie
info: System.Net.Http.HttpClient.api.ClientHandler[100]
Sending HTTP request GET https://api.example-environment.architect-ryan.dev.arcitect.io/movie
fail: Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware[1]
An unhandled exception has occurred while executing the request.
System.Net.Http.HttpRequestException: The SSL connection could not be established, see inner exception.
---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid because of errors in the certificate chain: NotTimeValid, UntrustedRoot
at System.Net.Security.SslStream.SendAuthResetSignal(ProtocolToken message, ExceptionDispatchInfo exception)
at System.Net.Security.SslStream.CompleteHandshake(SslAuthenticationOptions sslAuthenticationOptions)
at System.Net.Security.SslStream.ForceAuthenticationAsync[TIOAdapter](TIOAdapter adapter, Boolean receiveFirst, Byte[] reAuthenticationData, Boolean isApm)
at System.Net.Http.ConnectHelper.EstablishSslConnectionAsync(SslClientAuthenticationOptions sslOptions, HttpRequestMessage request, Boolean async, Stream stream, CancellationToken cancellationToken)
--- End of inner exception stack trace ---
at System.Net.Http.ConnectHelper.EstablishSslConnectionAsync(SslClientAuthenticationOptions sslOptions, HttpRequestMessage request, Boolean async, Stream stream, CancellationToken cancellationToken)
at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
at System.Net.Http.HttpConnectionPool.AddHttp11ConnectionAsync(HttpRequestMessage request)
at System.Threading.Tasks.TaskCompletionSourceWithCancellation`1.WaitWithCancellationAsync(CancellationToken cancellationToken)
at System.Net.Http.HttpConnectionPool.GetHttp11ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
at System.Net.Http.HttpConnectionPool.SendWithVersionDetectionAndRetryAsync(HttpRequestMessage request, Boolean async, Boolean doRequestAuth, CancellationToken cancellationToken)
at System.Net.Http.DiagnosticsHandler.SendAsyncCore(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
at Microsoft.Extensions.Http.Logging.LoggingHttpMessageHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
at Microsoft.Extensions.Http.Logging.LoggingScopeHttpMessageHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
at System.Net.Http.HttpClient.g__Core|83_0(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationTokenSource cts, Boolean disposeCts, CancellationTokenSource pendingRequestsCts, CancellationToken originalCancellationToken)
at asp_net_core_web.Pages.IndexModel.OnGet() in /app/asp-net-core-web/Pages/Index.cshtml.cs:line 42
at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.ExecutorFactory.NonGenericTaskHandlerMethod.Execute(Object receiver, Object[] arguments)
at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.InvokeHandlerMethodAsync()
at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.InvokeNextPageFilterAsync()
at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.Rethrow(PageHandlerExecutedContext context)
at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.InvokeInnerFilterAsync()
at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.g__Awaited|25_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
at Microsoft.AspNetCore.Routing.EndpointMiddleware.g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
```

**Comment by mueschm** [comment]
1. Liveness probe is in. Added to others but forget here so good catch.
2. Removed the item so it does not show up.
3. You are seeing that because ASP.net by default requires a valid SSL cert to run. You can disable that but as our default behavior is to add SSL certs to everything it made no sense. I assume you deployed to staging or local since my production deploy works fine.

**Review by ryan-cahill — APPROVED** [review]
`architect dev`
image

`architect dev --debug=false`
image

`architect deploy asp-net-core -e example-environment --auto-approve`
image

**Review by zach-j-dev — APPROVED** [review]
Looks good from my end! Everything runs smoothly and works as expected

**Pull request merged**